profiles/common/modules/custom/ecas/includes/ecas.inc

```diff
@@ -180,10 +184,20 @@ function ecas_menu_check() {
  * Redirects the user to the account request page.
  */
 function account_request() {
+  $account_request_url = ecas_get_account_request_url();
+  drupal_goto($account_request_url);
+}
+
+/**
+ * Gets the URL for the EU Login account creation form.
+ *
+ * @return string
+ *   The creation form URL.
+ */
+function ecas_get_account_request_url() {
```

Comment:
It's an internal function, it should be prefixed with "_".

Reply:
I was not planning to make it private because I do not see why it could not be part of the module API.
Is this problematic?